src/app/classes/category.ts

```diff
@@ -0,0 +1,6 @@
+export class Category {
+  name: string = '';
+  id: string = '';
+  tasks?: string []
+  constructor() {}
+}
```

Comment:
Conventionally, we don't use classes when defining data types for Angular apps. Especially in this case where nothing is instantiated.

Use interfaces instead and manage initial values in components, services, etc. that are using them.